plugins/autocast/rollup.config.mjs

```diff
@@ -22,7 +22,7 @@ function commonPlugins(browser, umd = false) {
     json(),
     commonjs({
       include: '**/node_modules/**',
-      requireReturnsDefault: 'preferred',
+      requireReturnsDefault: 'auto',
```

Comment:
We should make a shared Rollup config so that 1 change doesn't hit every package!

Reply:
I like the idea of drying the Rollup config, but don't we want it to hit every package to trigger changset?